src/components/table/demo/filtered/index.tsx

```diff
@@ -67,7 +67,7 @@ export class TableDemoFiltered implements ComponentWillLoad {
 						toolTip="Colored cats"
 						slot="bar"
 					/>
-					<smoothly-filter-picker label="coat" property="nested.coat" slot="bar" multiple={false}>
+					<smoothly-filter-picker label="coat" property="nested.coat" slot="bar" multiple={false} looks="border">
```

Comment:
i think it should be the one without border

Reply:
But now there is no border on the ones in the expand thingy...
Also the border around the search bar dissappeared.. but idk if it should have it